api_core/tests/unit/gapic/test_method.py

```diff
@@ -94,6 +94,19 @@ def test_invoke_wrapped_method_with_metadata():
     assert ('a', 'b') in metadata
 
 
+def test_invoke_wrapped_method_with_metadata_as_none():
+    method = mock.Mock(spec=['__call__'])
+
+    wrapped_method = google.api_core.gapic_v1.method.wrap_method(method)
+
+    wrapped_method(mock.sentinel.request, metadata=None)
+
+    method.assert_called_once_with(mock.sentinel.request, metadata=mock.ANY)
```

Comment:
Why use `mock.ANY` here?

Reply:
Because I don't need the tests to re-calculate the metrics header.

Reply:
Ah